### PR TITLE
yo dawg I hear you like whitespace fixing, so I fixed your whitespace in your whitespace fixer

### PR DESCRIPTION
**Pull request by magopian**
Awesome idea, truly awesome!

I've taken on the daunting task (ok, I might be exaggerating a bit ;) of fixing the whitespace and indentation in your bot.

While doing this, I found two bugs:
- line 160: there's no "c" anywhere, I guess it's meant to be "b"
- line 258: there's a line continuation missing and the beginning of the following line is missing the string start

Hope this helps!

Mathieu

### CODE REVIEW

**Comment by magopian** [comment]
Oh and btw, I would add a old_users.txt placeholder file, just for good measure, as without it (and without the -o option) the bot complains.

File: robot.py [hunk 1]
```diff
@@ -89,94 +90,100 @@ def main():
...
     repo = os.path.join(settings.pwd, repo)
     for root, dirs, files in os.walk(repo):
-       for f in files:
-        path = os.path.join(root, f)
-
-        # gotta be a way more pythonic way of doing this
-        banned = ['.git', '.py', '.yaml', '.patch', '.hs', '.occ']
-        cont = False 
-        for b in banned:
-            if c in path:
```
Comment (magopian) [inline robot.py:160]: are you sure it's meant to be "c" here and not "b" ?

File: robot.py [hunk 2]
```diff
@@ -184,91 +191,97 @@ def fix_repo (repo):
...
             ' a huge deal, but it\'s a bit annoying if you use Vim in a terminal. Really, I\'m just a proof of ' +
             ' concept - GitHub\'s V3 API allows robots to automatically improve open source projects, and that\'s really' +
-            ' cool. Hopefully, somebody, maybe you!, will fork me and make me even more useful. My owner is '+
+            ' cool. Hopefully, somebody, maybe you!, will fork me and make me even more useful. My owner is ' +
             '[funding a bounty](http://gun.io/open/12/add-security-flaw-fixing-features-to-whitespacebot) to anybody ' +
             'who can add security fixing features to me. ' +
             '\n\nI\'ve only cleaned your most popular project, and I\'ve added you to a list of users not to contact ' +
             'again, so you won\'t get any more pull requests from me unless you ask. If I\'m misbehaving, please email my ' +
-            'owner and tell him to turn me off! If this is pull request is of no use to you, please just ignore it.\n\n
-            Thanks!\nWhiteSpacebot from [Gun.io](http://gun.io).',
```
Comment (magopian) [inline robot.py:259]: There's a line continuation issue here

**Comment by Miserlou** [comment]
You rule. Pulled!

**Pull request merged**